chat-commands.js

```diff
@@ -2292,21 +2294,21 @@ exports.commands = {
 			// Get a list of all the rooms
 			let fileList = fs.readdirSync('logs/modlog');
 			for (let i = 0; i < fileList.length; ++i) {
-				filename += path.normalize(__dirname + '/' + logPath + fileList[i]) + ' ';
+				filename += path.normalize(`${__dirname}/${logPath}${fileList[i]}`) + ' ';
```

Comment:
it's 11 characters (`${}${}${}`) with template strings and 5 characters without (+''++). The advantage of template strings is lost, because I'm connecting a lot of variables, without string literals in between.

About path.normalize, I'm not sure what can be done :(